name-service/js/yarn.lock

```diff
@@ -4,21 +4,21 @@
 
 "@babel/code-frame@7.12.11":
   version "7.12.11"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
+  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz"
```

Comment:
Could you regenerate the yarn file using yarn? Looks like everything got changed to registry.npmjs.org rather than registry.yarnpkg.com